backend/typescript/services/implementations/camperService.ts

```diff
@@ -197,6 +197,56 @@ class CamperService implements ICamperService {
       throw error;
     }
   }
+
+  async deleteCamperByCamperId(camperId: string): Promise<void> {
```

Comment:
the nitpickiest of nits: can we rename this to `deleteCamperById`? just to make it a bit more concise